exercises/audio tinkering 1/Music.py

```diff
@@ -0,0 +1,40 @@
+import pygame
+pygame.init()
+pygame.mixer.init()
+#set window dimensions and caption
+
+
+
+window = pygame.display.set_mode((800,600))
```

Comment:
Consider defining the tuple `(800, 600)` as a constant for ease of modification.